doc/user/content/sql/explain.md

```diff
@@ -249,8 +249,9 @@ Operator | Meaning | Example
 `EXPLAIN TIMESTAMP` displays the timestamps used for a `SELECT` statement, view, or materialized view -- valuable information to acknowledge query delays.
 
 The explanation divides in two parts:
-1. Determinations for a timestamp
-2. Sources frontiers
+
+* Determinations for a timestamp
+* Sources frontiers
```

Comment:
I'm not sure, but shouldn't this be "Source frontiers"? At least, grepping for "source frontier" turns up various code comments, but grepping for "sources frontier" turns up results only from this md.